app/js/Editor.js

```diff
@@ -122,6 +123,10 @@ class Editor {
       this.hideRightPanel()
       return
     }
+
+    if (filter.substring(0, 1) !== '.')
```

Comment:
This excludes a lot of `jq` commands such as `keys`, `map()`, `del()`, etc.

https://stedolan.github.io/jq/manual/#Basicfilters